### PR TITLE
Estimations

### PR DESCRIPTION
**Pull request by DavidVorick**
This adds a host count estimation to the UI.

Also cleans up the host capsule a bit.

### CODE REVIEW

File: site/scripts/controller/controller.js [hunk 1]
```diff
@@ -314,6 +314,7 @@ var controller = (function() {
     function updateRenter(callback) {
         $.getJSON(uiConfig.siad_addr + "/renter/status", function(response) {
             data.filePrice = response.Price;
+			data.hostCount = response.KnownHosts;
```
Comment (lukechampine) [inline site/scripts/controller/controller.js]: muh indenting

File: site/scripts/ui/views/files.js [hunk 2]
```diff
@@ -129,9 +130,10 @@ ui._files = (function(){
         eFiles = $(newFileElements);
     }
 
-	function updateFilePrice(filePrice){
-		eRenterPrice.text("Estimated Price Per GB: " + util.siacoin(filePrice).toFixed(5) + " KS");
-	}
+    function updateFilePrice(filePrice, hostCount){
+        eRenterPrice.text("Estimated Price Per GB: " + util.siacoin(filePrice).toFixed(5) + " KS");
+        eRenterHostCount.text("Known Hosts: " + hostCount);
+    }
```
Comment (lukechampine) [inline site/scripts/ui/views/files.js]: this function name is misleading now

**Comment by lukechampine** [comment]
does this need to be changed to report `Competition` rather than `KnownHosts`?

File: site/scripts/ui/views/files.js (hunk 2 above)
Comment (DavidVorick) [inline site/scripts/ui/views/files.js]: will fix

**Comment by DavidVorick** [comment]
KnownHosts is a metric for the renter, competition is a metric for the host. They each get reported in their respective views.

**Comment by lukechampine** [comment]
ah, right

**Comment by DavidVorick** [comment]
ok. With all these changes I think we're basically done with the minor version. I think we've got everything we need on both the daemon and the gui.

Maybe you have some last minute stuff you want to squeak through

**Pull request merged**